types/applepayjs/index.d.ts

```diff
@@ -965,6 +973,37 @@ declare namespace ApplePayJS {
         tokenNotificationURL?: string | undefined;
     }
 
+    /**
+     * The response for information about the device’s support for Apple Pay and the payment credential status.
+     */
+    interface PaymentCredentialStatusResponse {
+        /**
+         * The status of the possible payment credentials that a person has provisioned in Wallet.
+         */
+        paymentCredentialStatus: PaymentCredentialStatus,
+    }
+
+    /**
+     * Information about whether the device supports Apple Pay and the possible payment credentials the person has provisioned in Wallet.
+     */
+    type PaymentCredentialStatus =
+        /**
+         * Confirms that the device supports Apple Pay and there’s at least one active payment credential in Wallet that qualifies for payments on the web. Show an Apple Pay button and offer Apple Pay as the primary, but not necessarily sole, payment option.
+         */
+        | "paymentCredentialsAvailable"
```

Comment:
```suggestion
        | "paymentCredentialStatusAvailable"
```

Reply:
Are you sure about that? Documantation says clearly that this is `paymentCredentialsAvailable` status. Am I missing something?
![image](https://github.com/user-attachments/assets/abdcfc00-1b85-4d15-9948-73172b0da68a)

Reply:
Fun....

So, if you click on `[PaymentCredentialStatus](https://developer.apple.com/documentation/apple_pay_on_the_web/paymentcredentialstatus)` directly from [`applePayCapabilities`](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession/4440085-applepaycapabilities), it takes you to [this document](https://developer.apple.com/documentation/apple_pay_on_the_web/paymentcredentialstatus) which states `paymentCredentialsAvailable`.

However, if you click on `PaymentCredentialStatusResponse` which takes you to [`PaymentCredentialStatusResponse`](https://developer.apple.com/documentation/apple_pay_on_the_web/paymentcredentialstatusresponse), then click on `paymentCredentialStatus`, it takes you to [`paymentCredentialStatus`](https://developer.apple.com/documentation/apple_pay_on_the_web/paymentcredentialstatusresponse/4440088-paymentcredentialstatus) which states `paymentCredentialStatusAvailable`.

![{F49AB32F-53A6-4D55-ABAA-722037A78DB3}](https://github.com/user-attachments/assets/79f719c8-c346-4d63-9d09-60fa47525da3)

Reply:
Well, there is some inconsistency in the documentation then. I just checked actual result and it seems that correct value is `paymentCredentialsAvailable`.
<img width="437" alt="image" src="https://github.com/user-attachments/assets/a27d738c-ba57-4371-8e79-7c341eace852">
I just submitted bug report to Apple about that.